### PR TITLE
Hotfix: Resolves address issues.

### PR DESCRIPTION
**Pull request by shshamim63**
### PR resolves the following issues:
___
Issue 1: Refactoring large methods -> #4 
Issue 2: Unnecessary deviation from standard format for controller actions -> #3
Issue 3: The Payment model is not really storing records of payments -> #2 
___
**Test**
___
Added model test for Loan and Payment model.

### CODE REVIEW

File: app/models/payment.rb [hunk 1]
```diff
@@ -1,43 +1,35 @@
 class Payment < ApplicationRecord
```
Comment (Vignesh1115) [inline app/models/payment.rb]: @shshamim63 Can you think of ways to improve the responsibility of the `Payment` class?

File: app/controllers/loans_controller.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  def create
+    @loan = Loan.create(loan_params)
+    if @loan.save
+      redirect_to @loan
+    end
+  end
+
+  def show
+    @loan = Loan.new
```
Comment (Vignesh1115) [inline app/controllers/loans_controller.rb:15]: Why would we need both `@loan` and `@seted_loan` here?

Comment (shshamim63) [reply app/controllers/loans_controller.rb:15]: The view loans/show is rendering a partial form and the calculated payments based on the id that was created just before. So `@loan` is needed for the model instance and `@seted_loan` is needed to get the information about the created loan.

File: app/models/payment.rb (hunk 1 above)
Comment (shshamim63) [reply app/models/payment.rb]: Yes, I think a separate service is a more feasible way for this task.

**Pull request merged**